README.md

```diff
@@ -1 +1,11 @@
-# ursa-docs
\ No newline at end of file
+# Documentation for Ursa
+
+This repository is for documentation, specifications, and anything else for 
+[ursa](https://github.com/hyperledger/ursa).
+
+Documents are broken down into the following folders:
+
+- [docs](docs/README.md) is used for documenting the APIs and how to use the ursa library. Ursa uses [mdBook](https://rust-lang-nursery.github.io/mdBook/) for now. 
```

Comment:
README.md doesn't exist in either directory. Did you mean to add one or is this a copy paste error?

Reply:
I plan to add them with my next PR.